docs/sources/shared/flow/reference/components/otelcol-filter-resource-block.md

```diff
@@ -12,14 +12,14 @@ headless: true
 This block specifies items to match the resources against:
 
 * More than one `resource` block can be defined.
-* A match occurs if the input data resources matches at least one `resource` block.
+* A match occurs if the input data resources match at least one `resource` block.
 
 The following arguments are supported:
 
-Name | Type | Description | Default | Required
----- | ---- | ----------- | ------- | --------
-`key` | `string` | The resource key. | | yes
-`value` | `any` | The resource value to match against. | | no
+Name    | Type     | Description                          | Default | Required
+--------|----------|--------------------------------------|---------|---------
+`key`   | `string` | The resource key.                    |         | yes
+`value` | `any`    | The resource value to match against. |         | no
 
 If `value` is not set, any value will match.
-The type of `value` could be a number, a string or a boolean.
+The type of `value` could be a number, a string, or a Boolean.
```

Comment:
I'm not sure I understand the capitalization of boolean here; it feels weird to me when next to other uncapitalized type names (number, string). (Though I do acknowledge my editor is currently complaining that "boolean" is misspelled and needs to be capitalized).

I'd like for the casing to be consistent when referring to basic type names, and ideally for it to be lowercase since that feels more common; should we rename all instances of "boolean" to "bool" to allow it to be lowercase?

Reply:
@rfratto Yeah good point. I am used to using uppercase in all instances, but after some digging,  https://developers.google.com/style/word-list#boolean suggests using lowercase for the data type.